test_mpop_service/TestMpopService.cpp

```diff
@@ -547,6 +547,7 @@ void TestMpopService::test_24_getAnswerByGender() {
     QMap<QString,int> answers = this->facade->getAnswerByGender(TEST_QUESTION_07_ID,"all",-1,-1,"all");
 
     QCOMPARE(answers[TEST_USER_1_GENDER], Test_QUESTION_07_AVG);
+
```

Comment:
Ideally, we'd want to avoid white space changes that don't follow our standards.